### DISCUSSION TITLE
This is an experiment: How to close a discussion

### DISCUSSION THREAD

**Discussion by GrampaWildWilly** [aclap-dev/video-downloadhelper#24] [Help and Q&A]
### Your question

I've been discovering that I can't close a discussion I didn't open.  So I'm opening a discussion to see exactly what I have to do.

### Addon & CoApp version and details

For the record, not that it's actually relevant to this experiment, I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3085, Firefox 122.0 64-bit, VDH 8.2.0.18a1 beta, CoApp 2.0.14.

**Comment by GrampaWildWilly** [comment]
OK.  Now that I have this open, I want to see how to close it, including how to mark a comment as the answer.

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/b6b3fb07-656a-45e3-8dab-76a6752e012e)

OK.  Let's see what happens when I actually click the first entry in that little list.

Well, actually, nothing changes.  All right.  Now I'm going to click the green Comment button.

**Comment by GrampaWildWilly** [comment]
OK.  That didn't actually close this.  But this little bit showed up under the preceding answer after I posted it:

![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/b7ceaa00-0909-4f1e-b6d7-4bb9e03ba7eb)

Doesn't take a degree in information technology to figure out I need to click that button to mark the reply as the answer.  So after I post this, I will click that button.

**Comment by GrampaWildWilly** [comment]
Oh.  That changes several things in the thread.  I won't point them out.  You can just scroll up & down & see them.  So now I'm going to actually try to close this.

**Comment by GrampaWildWilly** [comment]
It appears that AFTER you select a comment as the answer, you can click the button that says "Close with comment" & that closes the discussion.  Don't click the little triangle, click the words.

OK.  So that's how it works.  And it appears that only the original poster in a discussion can do this.  And of course anybody with Wizard Powers (like for example Paul).  I definitely don't have Wizard Powers.  I don't see the button for marking a comment as the answer in threads started by other people.  So I can't make such threads show me the button that will close the discussion.  It appears that other than Wizards, only the original poster in a thread can close it.

I find this all good to know.  I hope others do as well.

**Comment by GrampaWildWilly** [comment]
Now that I've closed this discussion, I'm seeing this at the bottom of the page:

![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/0365db6d-48ce-48db-a462-c2b10a0ad2fa)

I don't see that in closed discussions that I didn't open.  So closing a discussion isn't irrevocable.

On the topic of closed discussions, the default display for the list of discussions looks like this:

![#04](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/5b292939-a89c-40fb-830f-55ae88ca8277)

If you want to see closed discussions, & I strongly suggest you do, you have to do this:

![#05](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/915fd4b9-d0cb-4c4f-9528-1ecbc47245be)

**Comment by GrampaWildWilly** [comment]
Now this is hilarious.  Because I marked my own answer in this discussion as the answer to this question, I am now listed as Most Helpful.

![#06](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/47819414-5293-46b2-a0bb-f5e0a1e4f947)

I suppose I should say that I can now die happy.

**Reply by paulrouget** [reply]
🥇 🥇 🥇  :)

**Reply by GrampaWildWilly** [reply]
THREE medals . . . Watch it.  I might get a swelled head . . .
